### PR TITLE
Fix: remove hotbar name from status bar

### PR DESCRIPTION
**Pull request by aleksfront**
* Reverting part of the changes from https://github.com/lensapp/lens/pull/4476 by removing hotbar name from status bar
* Showing active hotbar name in tooltip when switching hotbars

https://user-images.githubusercontent.com/9607060/149136446-7ef8efcd-0412-4a65-a2ab-e4ff7da62843.mov

### CODE REVIEW

File: src/renderer/components/hotbar/hotbar-selector.tsx [hunk 1]
```diff
@@ -45,25 +46,59 @@ interface Dependencies {
...
-        id="hotbarIndex"
-        small
-        label={hotbarManager.getDisplayIndex(hotbarManager.getActive())}
-        onClick={() => openCommandOverlay(<HotbarSwitchCommand />)}
-        tooltip={{
-          preferredPositions: [TooltipPosition.TOP, TooltipPosition.TOP_LEFT],
-          children: hotbar.name,
-        }}
-        className="SelectorIndex"
+const NonInjectedHotbarSelector = observer(({ hotbar, hotbarManager, openCommandOverlay }: HotbarSelectorProps & Dependencies) => {
```
Comment (ixrock) [inline src/renderer/components/hotbar/hotbar-selector.tsx:33]: Name `NonInjectedHotbarSelector` imply that it props interface for non-injected component should be:

```suggestion
export interface HotbarSelectorProps extends Partial<Dependencies> {...}

const NonInjectedHotbarSelector = observer(({ hotbar, hotbarManager, openCommandOverlay }: HotbarSelectorProps) => {
```

File: src/renderer/components/hotbar/hotbar-selector.tsx [hunk 2]
```diff
@@ -45,25 +46,59 @@ interface Dependencies {
...
+  const onTooltipShow = () => {
+    setTooltipVisible(true);
+    clearTimer();
+    tooltipTimeout.current = setTimeout(() => setTooltipVisible(false), 1500);
+  };
+
+  const onArrowClick = (switchTo: () => void) => {
+    onTooltipShow();
+    switchTo();
+  };
```
Comment (ixrock) [inline src/renderer/components/hotbar/hotbar-selector.tsx]: I would recommend to use named functions instead (no need in keeping `this` context) + better readability / debugging / hoisting out-of-the box.

File: src/renderer/components/tooltip/tooltip.tsx [hunk 3]
```diff
@@ -226,9 +230,9 @@ export class Tooltip extends React.Component<TooltipProps> {
   }
 
   render() {
-    const { style, formatters, usePortal, children } = this.props;
+    const { style, formatters, usePortal, children, visible } = this.props;
     const className = cssNames("Tooltip", this.props.className, formatters, this.activePosition, {
-      invisible: !this.isVisible,
+      invisible: visible === undefined ? !this.isVisible : !visible,
```
Comment (ixrock) [inline src/renderer/components/tooltip/tooltip.tsx]:
```suggestion
      visible: this.props.visible ?? this.isVisible,
```

**Review by ixrock — DISMISSED** [review]
Almost LGTM

**Review by jweak — APPROVED** [review]
LGTM!

**Review by ixrock — APPROVED** [review]
LGTM

**Pull request merged**